apps/dashboard/src/helpers/getBackgroundUrl.js

```diff
@@ -27,7 +27,9 @@ import prefixWithBaseUrl from './prefixWithBaseUrl'
 
 export default (background, time = 0, themingDefaultBackground = '') => {
 	const enabledThemes = window.OCA.Theming.enabledThemes
-	const isDarkTheme = enabledThemes.join('').indexOf('dark') !== -1
+	const isDarkTheme = enabledThemes && (enabledThemes.length > 1 || enabledThemes[0] !== 'default')
+		? enabledThemes.join('').indexOf('dark') !== -1
+		: window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
```

Comment:
Again, could this work ?

```suggestion
const isDarkTheme = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
```

Reply:
Adjusted this like in #33299